docs/data-guidelines.md

```diff
@@ -177,6 +177,12 @@ However, this guideline does not apply to features where the browser's expected
 
 This guideline was proposed in [#6906](https://github.com/mdn/browser-compat-data/issues/6906).
 
+## Constants
+
+We don't include data for constants in BCD. The rationale for not including them is that they're not known to be a source of any compatibility issues.
+
```

Comment:
It'd be nice to have a concrete example of a constant that we would omit from BCD.

Reply:
Agreed. Will add

Reply:
> It'd be nice to have a concrete example of a constant that we would omit from BCD.

OK, added